### ISSUE TITLE
Date articles added to DOAJ

### ISSUE THREAD

**Issue by dommitchell** [DOAJ/doaj#711]
Do you have any way of telling when two articles were added to DOAJ?

https://doaj.org/search?source={%22query%22%3A{%22query_string%22%3A{%22query%22%3A%22\%22Reproductive%20Biology%20of%20Bothrops%20erythromelas%20from%20the%20Brazilian%20Caatinga\%22%22%2C%22default_operator%22%3A%22AND%22}}%2C%22from%22%3A0%2C%22size%22%3A10}

**Comment by emanuil-tolev** [comment]
2nd April 2015 via upload.

**Comment by dommitchell** [comment]
Thanks! Have we disabled DOI matching?

**Comment by emanuil-tolev** [comment]
Hmm. Not that I'm aware of. Yeah, on both bibjson.identifier contains a type "doi" key with value "10.1155/2014/680861" (no spaces or other wonky things going on).

**Comment by dommitchell** [comment]
@richard-jones do you know why matching didn't happen on these articles?

**Comment by richard-jones** [comment]
These are the dates these were added/updated

last_updated: "2015-04-22T01:25:21Z",
created_date: "2015-04-02T08:00:15Z"

and

last_updated: "2015-04-22T02:27:21Z",
created_date: "2015-04-02T10:00:08Z"

I've done some investigation, and there's no doubt that the matching algorithm is working correctly - I was able to do a mock match for one of these and found the other.

I have also check the article_history index to see if there are any clues, and there are not.

I hate to give up, but investigation and testing today have yielded no insights into this whatsoever, and it might have to just go down as an oddity.  If we see it again we could have another look, and if we can find a reproducible example, that would make our lives easier.

Anyone else got any ideas?
